cmd/aws/main.go

```diff
@@ -230,3 +236,32 @@ func storageConfigFromFlags() taws.Config {
 		MaxIdleConns: *dbMaxIdle,
 	}
 }
+
+func antispamMysqlConfig() *mysql.Config {
```

Comment:
```suggestion
func antispamMySQLConfig() *mysql.Config {
```

https://google.github.io/styleguide/go/decisions#initialisms